webpack/assets/javascripts/react_app/components/Editor/EditorHelpers.js

```diff
@@ -0,0 +1,156 @@
+import {
+  isEmpty,
+  filter,
+  find,
+  replace,
+  capitalize,
+  forEach,
+  concat,
+  sortBy,
+} from 'lodash';
+
+import { setCompleters } from 'ace-builds/src-min-noconflict/ext-language_tools';
```

Comment:
As of https://github.com/theforeman/foreman-js/pull/363/files#diff-49c3b2f1af7619149c2b3f68c0fd26f135c1dcc02932cf76f510da25c69bebc5R65 (don't ask me why ^_^)
```suggestion
import { setCompleters } from 'ace-builds/src-noconflict/ext-language_tools';
```

Reply:
My bad :facepalm:, updated.